_opensearch/install/tar.md

```diff
@@ -453,6 +453,86 @@ $ curl https://your.host.address:9200 -u admin:yournewpassword -k
 }
 ```
 
+### Run OpenSearch as a service with systemd
+
+Create a service for OpenSearch and register it with `systemd`. After the service has been defined, you can enable, start, and stop the OpenSearch service using `systemctl` commands. The commands in this section reflect an environment where OpenSearch has been installed to `/opt/opensearch` and should be changed depending on your installation path.
```

Comment:
This makes it sound like the user should have added the systemd config *before* starting this section. Maybe something like "To manage OpenSearch as a service, you will need to register it with `systemd`.

Reply:
I see what you mean. I saw the approvals before the comments so didn't make this change but I'll open a new PR. Good call out.